specs/scoring_spec.rb

```diff
@@ -42,21 +42,36 @@
 
   describe 'highest_score_from' do
     it 'returns nil if no words were passed' do
+      Scrabble::Scoring.highest_score_from([]).must_be_nil
     end
 
     it 'returns the only word in a length-1 array' do
+      words = ['academy']
+      Scrabble::Scoring.highest_score_from(words).must_equal 'academy'
     end
 
     it 'returns the highest word if there are two words' do
+      words = ['cat', 'house']
+      Scrabble::Scoring.highest_score_from(words).must_equal 'house'
     end
```

Comment:
Like we discussed in class, it would probably be wise to check multiple orderings of these words, that is, both `['dog', 'fish']` and `['fish', 'dog']`. You'd be surprised how often this sort of thing turns up a bug.